data/modules/index.js

```diff
@@ -0,0 +1,538 @@
+"use strict";
```

Comment:
was this file supposed to be a part of this commit? :thinking:

Reply:
good catch, removed.